dask/verify_dask_standalone.py

```diff
@@ -1,11 +1,6 @@
-from dask.distributed import Client
 import dask.array as da
-import dask_ml
-import dask_bigquery
 
 import numpy as np
 
-client = Client("localhost:8786")
```

Comment:
Can you convince me that we don't care about dask.distributed ?

Reply:
This variable is not being used in the script. It is only declared.

Reply:
The process of instantiating the object is the test.  If an exception is thrown, the test is a FAIL as it should be